name.abuchen.portfolio/src/name/abuchen/portfolio/online/SecuritySearchProvider.java

```diff
@@ -46,7 +51,8 @@ default boolean hasPrices()
 
     public enum Type
     {
-        ALL(Messages.LabelSearchAll), SHARE(Messages.LabelSearchShare), BOND(Messages.LabelSearchBond);
+        ALL(Messages.LabelSearchAll), SHARE(Messages.LabelSearchShare), BOND(Messages.LabelSearchBond), //
+        ETC(Messages.LabelSearchETC), ETF(Messages.LabelSearchETF), FUND(Messages.LabelSearchFund);
```

Comment:
I propose **not** to add additional labels ETC, ETF, FUND, etc.

First, it is not used to filter the resource anyway (see the screenshot where I picked to search for "ETF" but it returns everything. Also the Leeway implementation ignores these labels for searching (you can set an arbitrary type in the result if you just want to display it - continue to use the translations, but to not add the enum here).

<img width="1025" alt="Bildschirmfoto 2023-05-10 um 19 45 14" src="https://github.com/buchen/portfolio/assets/587976/04b4397b-7ab4-4483-b69a-6785ec624e59">

Second, if we add new types, we should at least support the primary quote feed "Portfolio Report". Otherwise users choose to search for ETF and then are asked to sign on to Leeway. These providers come and go. I want to support primarily Portfolio Report where we can improve the backend as well.